neo/Core/Witness.py

```diff
@@ -12,37 +7,67 @@ class Witness(SerializableMixin):
     InvocationScript = None
     VerificationScript = None
 
-    def __init__(self, invocation_script=None, verification_script=None):
+    def __init__(self, invocation_script=bytearray(), verification_script=bytearray()):
```

Comment:
why make invocation_script use an empty bytearray as default?

Reply:
because `ToJson` and `Serialize` will fail without it when you initialise an empty `Witness`   while it should return 2 bytes representing 1) 0 length of invocation script and 2) 0 length of verification script

Reply:
The question is: does it work with that default args of empty bytearray? Else better check if none and then error out.

Reply:
Yeah it works with those default args. The `Witness` class is covered 100% in the test file and once I merge the commits from #123 it will show.